h/static/styles/partials-v2/_form-input.scss

```diff
@@ -80,16 +80,31 @@
     right: $h-padding;
 
     color: $grey-5;
- }
+  }
+
+  .form-input__hint-icon {
+    display: none;
+    position: absolute;
+    top: 10px;
+    left: 55px;
+    cursor: help;
```

Comment:
The `help` cursor matches the spec, but on my Ubuntu system this unfortunately shows a large '?' cursor which partially obscures the label. I would be inclined to change it to just an arrow. We can address this after the merge.